content/02.main-text.md

```diff
@@ -1,32 +1,29 @@
+Our review manuscript [@doi:10.1098/rsif.2017.0387], code-named the Deep Review, surveyed deep learning's role in biology and precision medicine, a research area undergoing explosive growth.
+In addition, we introduce the infrastructure we created for Deep Review, termed [Manubot](https://github.com/greenelab/manubot-rootstock), that enables open manuscript writing online.
 
-We initiated our open online review article by creating a GitHub repository (https://github.com/greenelab/deep-review) to coordinate and manage contributions.
+We initiated the Deep Review by creating a GitHub repository (https://github.com/greenelab/deep-review) to coordinate and manage contributions.
 GitHub is a platform designed for collaborative software development that's adaptable for collaborative writing.
 From the start, we made the GitHub repository public, applying a [Creative Commons Attribution License](https://github.com/greenelab/deep-review/blob/master/LICENSE.md) to the manuscript.
 Next, we encouraged anyone interested to contribute by proposing changes or additions.
-Although we requested that some authors participate for their specific expertise, most discovered the manuscript organically through conferences or social media, deciding without solicitation to contribute.
-In total, the Deep Review attracted {{deep_review_authors}} authors from 20 different institutions (and counting) who were not determined in advance.
+Although we invited some specific experts to participate, most authors discovered the manuscript organically through conferences or social media, deciding to contribute without solicitation.
+In total, the Deep Review attracted {{deep_review_authors}} authors who were not determined in advance from 20 different institutions.
 
 Writing review articles in a public forum allows review authors to engage with the original researchers to clarify their methods and results and present them accurately, as exemplified [here](https://github.com/greenelab/deep-review/issues/213).
-Additionally, discussing manuscripts in the open generates valuable post-publication peer review [@doi:10.1016/j.tig.2015.03.006; @doi:10.1371/journal.pmed.1001772; @doi:10.3389/fncom.2012.00063] (and potentially pre-publication peer review in the case of preprints [@doi:10.7554/eLife.38532]).
-Since incentives are commonly lacking to provide public peer review of existing literature [@doi:10.1629/uksg.245], open collaborative reviews --- where authorship is open to anyone who makes a valid contribution --- could help spur more post-publication peer review.
+Additionally, discussing manuscripts in the open generates valuable pre-publication [@doi:10.7554/eLife.38532] and post-publication peer review [@doi:10.1016/j.tig.2015.03.006; @doi:10.1371/journal.pmed.1001772; @doi:10.3389/fncom.2012.00063].
```

Comment:
You're right, someone would have to infer that's what "pre-publication" means or go to the reference title to see that it pertains to preprints.  Is that okay for the sake of the shorter phrasing?

Reply:
I am okay with the current phrasing or something like the original:

> Additionally, discussing manuscripts in the open generates valuable post-publication peer review [@doi:10.1016/j.tig.2015.03.006; @doi:10.1371/journal.pmed.1001772; @doi:10.3389/fncom.2012.00063] or potentially pre-publication peer review in the case of preprints [@doi:10.7554/eLife.38532].

The reason I want to stress preprints is since there is currently great excitement towards generating preprints reviews and I think most people will overlook the connection unless we're explicit. What does @cgreene think?

Reply:
I think there is a pretty strong interest in the preprint ecosystem at the moment. What about:

> Additionally, authoring manuscripts in the open generates valuable feedback before a preprint is available, when a preprint is available [@doi:10.7554/eLife.38532] and after a paper is published [@doi:10.1016/j.tig.2015.03.006; @doi:10.1371/journal.pmed.1001772; @doi:10.3389/fncom.2012.00063].

I think there's something to be said for generating pre-prepublication peer review 😆 . In this system it can happen long before a preprint exists.

Reply:
I'm convinced, we should mention preprints explicitly.

@cgreene that phrasing combines two ideas: feedback on the manuscript being written and pre-/post-publication review of other manuscripts.  The first idea might fit better with the previous sentence about how open writing improves quality of the manuscript being written.  For the second idea, a version of @dhimmel's phrasing would be
> Additionally, discussing manuscripts in the open generates valuable post-publication peer review [@doi:10.1016/j.tig.2015.03.006; @doi:10.1371/journal.pmed.1001772; @doi:10.3389/fncom.2012.00063] or pre-publication peer review of preprints [@doi:10.7554/eLife.38532].

I like keeping the phrase `post-publication peer review`.

Reply:
👍

Reply:
> I like keeping the phrase post-publication peer review.

I agree. This is the term most people will be familiar with.